js/components/TutorialPanel.jsx

```diff
@@ -0,0 +1,59 @@
+var BootstrapModalMixin = require('../mixins/BootstrapModalMixin.jsx');
+var strings = require('../lib/strings_utils.js');
+var React = require('react');
+
+var TutorialPanel = React.createClass({
+  mixins: [BootstrapModalMixin],
+  propTypes: {
+    title: React.PropTypes.string,
+    content: React.PropTypes.string,
+    clazz: React.PropTypes.string,
+  },
+
+  getInitialState: function() {
+    return {
+      step: 0,
+    };
+  },
+
+  goToNext: function() {
```

Comment:
I think it's sort of weird that we had the tutorial manager, which starts the tutorial and "manages" the tutorial, but it doesn't continue to get the tutorial to the next step. It's more like the thing that initializes the tutorial.

Not saying that holds back this PR, but I think it is kind of confusing.